test/functional/p2p_invalid_tx.py

```diff
@@ -47,7 +47,7 @@ def reconnect_p2p(self, **kwargs):
         self.nodes[0].disconnect_p2ps()
         self.bootstrap_p2p(**kwargs)
 
-    def new_spend_tx(self, prev_hash, prev_n, values):
+    def new_spend_tx(self, prev_hash, prev_n, values, script):
```

Comment:
why?
This argument is not being used inside the function.

If we want to introduce (and use) it, I think it would be better to generalize it, so different outputs can have different scripts. In other words, turn the `values` list in a `CTxOut` list.

Reply:
done